src/test/java/PropositionalFormulaParserTest.java

```diff
@@ -0,0 +1,75 @@
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.prop4j.And;
+import org.prop4j.Literal;
+import org.prop4j.Node;
+import org.prop4j.Or;
+import org.variantsync.diffdetective.error.UnparseableFormulaException;
+import org.variantsync.diffdetective.feature.CPPAnnotationParser;
+
+import java.util.List;
+
+import static org.variantsync.diffdetective.util.Assert.assertEquals;
+
+public class PropositionalFormulaParserTest {
```

Comment:
Please add a short documentation (1-2 sentences) about this tests goals. Also you may add your name as an `@author` if you like. :)